packages/docs-app/src/examples/core-examples/cardListExample.tsx

```diff
@@ -0,0 +1,160 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+
+import { Button, Card, CardList, Classes, Code, H5, Section, SectionCard, Switch } from "@blueprintjs/core";
+import { Example, ExampleProps, handleBooleanChange } from "@blueprintjs/docs-theme";
+import { ChevronRight } from "@blueprintjs/icons";
+
+import { PropCodeTooltip } from "../../common/propCodeTooltip";
+
+export interface CardListExampleState {
+    isBordered: boolean;
+    isCompact: boolean;
+    useInteractiveCards: boolean;
+    useScrollableContainer: boolean;
+    useSectionCardPadding: boolean;
+    useSectionContainer: boolean;
+}
+
+export class CardListExample extends React.PureComponent<ExampleProps, CardListExampleState> {
+    public state: CardListExampleState = {
+        isBordered: true,
+        isCompact: false,
+        useInteractiveCards: true,
+        useScrollableContainer: false,
+        useSectionCardPadding: false,
+        useSectionContainer: false,
+    };
+
+    private get isBordered() {
+        return this.state.useSectionContainer ? this.state.useSectionCardPadding : this.state.isBordered;
+    }
+
+    public render() {
+        const { isCompact, useInteractiveCards, useScrollableContainer, useSectionCardPadding, useSectionContainer } =
+            this.state;
+
+        const options = (
+            <>
+                <H5>CardList Props</H5>
+                <PropCodeTooltip
+                    disabled={!useSectionContainer}
+                    content={
+                        <span>
+                            This example overrides <Code>isBordered</Code> when using a <Code>Section</Code> container
+                        </span>
+                    }
+                >
+                    <Switch
+                        checked={this.isBordered}
+                        disabled={useSectionContainer}
+                        label="Bordered"
+                        onChange={this.toggleIsBordered}
+                    />
+                </PropCodeTooltip>
+                <Switch checked={isCompact} label="Compact" onChange={this.toggleIsCompact} />
+                <H5>Card Props</H5>
+                <Switch checked={useInteractiveCards} label="Interactive" onChange={this.toggleUseInteractiveCards} />
+                <H5>Layout</H5>
+                <Switch
+                    checked={useSectionContainer}
+                    labelElement={
+                        <span>
+                            Use <Code>Section</Code> container
+                        </span>
+                    }
+                    onChange={this.toggleUseSectionContainer}
+                />
+                <H5 className={classNames({ [Classes.TEXT_MUTED]: !useSectionContainer })}>SectionCard</H5>
```

Comment:
this modifier currently has no effect, but I'm fixing that in a separate PR (I was surprised it didn't work): https://github.com/palantir/blueprint/pull/6304